src/openvslam/tracking_module.h

```diff
@@ -253,6 +261,15 @@ class tracking_module {
 
     //! Pause of the tracking module is requested or not
     bool pause_is_requested_ = false;
+
+    //! Update into a given position is requested or not
+    bool update_pose_is_requested();
+    //! Mutex for update pose into a given position
+    mutable std::mutex mtx_update_pose_;
+    //! Indicator of update pose request
+    bool update_pose_is_requested_ = false;
+    //! Requested pose to update
+    Mat44_t request_pose_;
```

Comment:
Would you prefer the following?

```suggestion
    Mat44_t requested_pose_;
```

Reply:
Done in latest update.